### DISCUSSION TITLE
Unable to generate Metrics Data

### DISCUSSION THREAD

**Discussion by kfhjack** [talmolab/sleap#1669] [Help!]
Hello everyone, I have predicted some videos, but I cannot generate Metrics Data，What should I do，The following is the error code

Thanks

D:\Miniforge3\virenvs\sleap\lib\site-packages\sleap\nn\evals.py:539: RuntimeWarning: Mean of empty slice
  "dist.avg": np.nanmean(dists),
D:\Miniforge3\virenvs\sleap\lib\site-packages\sleap\nn\evals.py:572: RuntimeWarning: Mean of empty slice.
  mPCK = mPCK_parts.mean()
D:\Miniforge3\virenvs\sleap\lib\site-packages\numpy\core\_methods.py:189: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
D:\Miniforge3\virenvs\sleap\lib\site-packages\sleap\nn\evals.py:666: RuntimeWarning: Mean of empty slice.
  pair_pck = metrics["pck.pcks"].mean(axis=-1).mean(axis=-1)
D:\Miniforge3\virenvs\sleap\lib\site-packages\numpy\core\_methods.py:182: RuntimeWarning: invalid value encountered in true_divide
  ret, rcount, out=ret, casting='unsafe', subok=False)
D:\Miniforge3\virenvs\sleap\lib\site-packages\sleap\nn\evals.py:668: RuntimeWarning: Mean of empty slice.
  metrics["oks.mOKS"] = pair_oks.mean()
WARNING:sleap.nn.evals:Failed to compute metrics.

**Comment by Marielamend** [comment]
Hello @kfhjack!
Thank you for your inquiry, 

A couple of follow-up questions to best assist you:

1. Did you get any predictions when you ran inference?
2. Do you recall what you selected to "Predict On:" under training pipeline? e.g.) suggested frames, user labeled frames, entire current video?

I also found a similar issue #511 from a while back that may help. 

Have a great day!,
Mariela
